cdap-ui/app/directives/widget-container/widget-radio-group/widget-radio-group.html

```diff
@@ -0,0 +1,39 @@
+<!--
+  Copyright © 2018 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+<div
+  ng-class="{
+    'widget-radio-group': true,
+    'widget-radio-group-inline': layout === 'inline'
+  }"
+>
+  <div
+    ng-class="{
+      radio: layout === 'block',
+      'radio-inline': layout === 'inline'
+    }"
+    ng-repeat="option in options">
+    <label>
+      <input
+        type="radio"
+        id="{{option.id}}"
+        value="{{option.id}}"
+        name="radio-group-buttons"
+        ng-model="selectionOption.value"
```

Comment:
can this simply be the `model`? That way we don't need the `$watch` on the controller